spec/models/event_attendance_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe EventAttendance, type: :model do
+  describe 'Create new eventattendance' do
+    u = User.new
+    u.save
+    e = Event.new(creator: u)
+    e.save
+    it 'Should work for everything is set correctly' do
+      a = EventAttendance.new(event_id: 1, attended_event_id: 1, attendee_id: 1)
+      expect(a.valid?).to eql(true)
+    end
```

Comment:
This part of the code is failing. Let us kindly address this. The error is "expected `true` but got `false`" :100: